extensions/wikia/AdEngine/js/SourcePoint.js

```diff
@@ -0,0 +1,41 @@
+/*global define*/
+/*jshint maxlen:125, camelcase:false, maxdepth:7*/
+define('ext.wikia.adEngine.sourcePoint', [
```

Comment:
I had this name before, but since here is shared client id I left just `sourcePoint` and renamed `init` method to `initDetection`, because this module is used in SourcePointTag :-)